implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/types/DynatraceSummary.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace.types;
+
+public class DynatraceSummary {
```

Comment:
If we don't want/need to subclass this, probably makes sense to mark it as `final`. Same for the other ones as well.

Reply:
I am not sure if we don't want to extend it with the Histogram in the future. Then we might want to create a subclass that also has Histogram support

Reply:
Exactly. If you mark final now, and want to extend later all is well. But if you _not_ mark now and want to _prevent_ it later, then we can't since it's a breaking change. Either is fine, just a heads up :)